kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/htm-applications.yaml

```diff
@@ -0,0 +1,25 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: htm-applications
+  annotations:
+    opendatahub.io/notebook-image-url:
+      "https://github.com/aicoe-aiops/htm-applications"
+    opendatahub.io/notebook-image-name:
+      "HTM Applications"
+    opendatahub.io/notebook-image-desc:
+      "Jupyter notebook image with htm package installed"
+  labels:
+    opendatahub.io/notebook-image: 'true'
+spec:
+  tags:
+    - name: latest
+      annotations:
+        openshift.io/imported-from: quay.io/aicoe/htm-applications
+      from:
+        kind: DockerImage
+        name: 'quay.io/aicoe/htm-applications:latest'
+      importPolicy: {}
```

Comment:
you can set the importPolicy to scheduled so it will update the imagestream automatically ([example](https://github.com/operate-first/apps/blob/3316164e4c0ebc7812d1463b7f4f175c12667558/kfdefs/overlays/osc/osc-cl1/jupyterhub/notebook-images/aicoe-osc-demo.yaml#L25-L26))

Reply:
thanks @4n4nd , I have updated the file.